lib/dynosaur/inputs/new_relic_input_plugin.rb

```diff
@@ -0,0 +1,62 @@
+require 'dynosaur/new_relic_api_client'
+require 'dynosaur/version'
+require 'dynosaur/error_handler'
+
+# ScalerPlugin implementation that uses the New Relic API
+# to get the current requests per minute on the site, and scale to an
+# appropriate number of dynos.
+
+module Dynosaur
+  module Inputs
+    class NewRelicInputPlugin < AbstractInputPlugin
+
+      # Load config from the config hash
+      def initialize(config)
+        super
+
+        @unit = "RPM"
+        @key = config["key"]
+        @app_id = config["app_id"].to_s
+        @rpm_per_dyno = config["rpm_per_dyno"].to_i
```

Comment:
Yeah, that'd be good, it seems like ga doesn't have a default either, want me to add one?